src/main/java/com/avail/descriptor/types/PrimitiveTypeDescriptor.java

```diff
@@ -317,44 +319,43 @@ public boolean o_IsSupertypeOfPojoBottomType (
 		return object.isSupertypeOfPrimitiveTypeEnum(NONTYPE);
 	}
 
-	@Override @AvailMethod
+	@Override
 	public boolean o_IsSupertypeOfPojoType (
 		final AvailObject object,
 		final A_Type aPojoType)
 	{
 		return object.isSupertypeOfPrimitiveTypeEnum(NONTYPE);
 	}
 
-	@Override @AvailMethod
+	@Override
 	public boolean o_IsSupertypeOfPrimitiveTypeEnum (
 		final AvailObject object,
 		final Types primitiveTypeEnum)
 	{
-		return
-			primitiveTypeEnum.superTests[extractOrdinal(object)];
+		return primitiveTypeEnum.superTests[extractOrdinal(object)];
 	}
 
-	@Override @AvailMethod
+	@Override
 	public boolean o_IsSupertypeOfSetType (
 		final AvailObject object,
-		final AvailObject aSetType)
+		final A_Type aSetType)
 	{
 		// This primitive type is a supertype of aSetType if and only if this
 		// primitive type is a supertype of NONTYPE.
 		return object.isSupertypeOfPrimitiveTypeEnum(NONTYPE);
 	}
 
-	@Override @AvailMethod
+	@Override
 	public boolean o_IsSupertypeOfTupleType (
 		final AvailObject object,
-		final AvailObject aTupleType)
+		final A_Type aTupleType)
 	{
 		// This primitive type is a supertype of aTupleType if and only if this
 		// primitive type is a supertype of NONTYPE.
 		return object.isSupertypeOfPrimitiveTypeEnum(NONTYPE);
 	}
 
-	@Override @AvailMethod
+	@Override
 	public boolean o_IsSupertypeOfEnumerationType (
 		final AvailObject object,
 		final A_BasicObject anEnumerationType)
```

Comment:
I believe this can be strengthened to A_Type.

Reply:
Requires a surprising number of updates, but yes, you are correct. Fixed.